pkg/data/values.go

```diff
@@ -1,5 +1,14 @@
+// Package data contains functions for working with unstructured values like interfaces or map[string]interface{}
+// allows reading/writing to these values without having to convert to structured items.
 package data
 
+import (
+	"strconv"
+)
+
+// RemoveValue removes a value from data. Keys should be in order of precedence (i.e. "metadata", "annotations") means
```

Comment:
```suggestion
// RemoveValue removes a value from data. Keys should be in order denoting the path to the value in the nested structure of the map.
// For example, passing []string{"metadata", "annotations"} will make the function remove the "annotations" key from the "metadata" sub-map.
```

Reply:
I've made some adjustments to the doc comment. LMK if you want any further changes.